Source/AwsCommonRuntimeKit/crt/Utilities.swift

```diff
@@ -121,6 +132,26 @@ extension TimeInterval {
     var millisecond: UInt64 {
         UInt64((self*1000).rounded())
     }
+
+    var millisecondUInt32: UInt32 {
+        UInt32((self*1000).rounded())
```

Comment:
I think we need a bounds check? I'm wondering if we need to do the same for the millisecond() as well since that doesn't check for negatives.